zmon-controller-app/src/main/java/org/zalando/zmon/api/SearchAPI.java

```diff
@@ -48,9 +59,32 @@
         List<QuickSearchResultItem> dashboards = searchService.quickSearchDashboards(search, teamList, limit);
         result.put("dashboards", dashboards);
 
-        List<QuickSearchResultItem> grafanaDashboards = searchService.quickSearchGrafanaDashboards(search, teamList, limit);
-        result.put("grafana_dashboards", grafanaDashboards);
+        result.put("grafana_dashboards", searchVisualizationDashboards(search, teams, limit, authHeader != null ? authHeader.split(" ")[1] : ""));
 
         return new ResponseEntity<>(result, HttpStatus.OK);
     }
+
+    private List<QuickSearchResultItem> searchVisualizationDashboards(String search, String teams, int limit, String token) {
+        List<QuickSearchResultItem> dashboards = new ArrayList<>();
+        Map<String, String> searchParams = new HashMap<>();
+        searchParams.put("query", search);
+        if (teams == null) {
+            searchParams.put("tag", "");
+        }
+        searchParams.put("tag", teams);
```

Comment:
why is tag set to team?

Reply:
This matches the existing feature while searching grafana dashboards: 
https://github.com/zalando-zmon/zmon-controller/blob/f02353d5604c17ed45368d72a0e9912619c39041/zmon-controller-app/src/main/resources/db/migration/V2__Initial_API.sql#L1473